install/kubernetes/cilium/charts/hubble-relay/values.yaml

```diff
@@ -39,17 +55,16 @@ sortBufferDrainTimeout: ~
 # Port to use for the k8s service backed by hubble-relay pods.
 # If not set, it is dynamically assigned to port 443 if TLS is enabled and to
 # port 80 if not.
-servicePort:
-
-# Specifies annotation for service accounts
-serviceAccount:
-  annotations: {}
-
-# Node labels for pod assignment
-# ref: https://kubernetes.io/docs/user-guide/node-selection/
-nodeSelector: {}
+servicePort: 80
 
-## Node tolerations for pod assignment on nodes with taints
-## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
-##
-tolerations: []
+tls:
+  enabled: true
```

Comment:
Could we move `enabled` under `client` ? Effectively, when set to `true` it enables TLS for the client only (ie: the part that connects to Hubble server instances). It would also be more consistent with `enabled` already in the `server` section.

Reply:
even better, we now rely on the hubble.tls.enabled flag for both ends after consolidating down to one chart